.travis.yml

```diff
@@ -38,8 +38,8 @@ before_install:
   - docker-compose -f ${TRAVIS_BUILD_DIR}/dev/docker/ci-compose.yml up -d db
 
   # Tweak git to correctly work with submodules.
-  - sed -i 's/git@github.com:/git:\/\/github.com\//' .gitmodules
-  - git submodule update --init --recursive
+  - git submodule update --init
```

Comment:
in TEC, these lines are added, not replacing the existing, so keep ` - sed -i 's/git@github.com:/git:\/\/github.com\//' .gitmodules` otherwise it will fail.